infrastructure-playbooks/purge-cluster.yml

```diff
@@ -423,18 +423,6 @@
 
   tasks:
 
-  - name: stop ceph.target with systemd
-    service:
-      name: ceph.target
-      state: stopped
-      enabled: no
-    when: ansible_service_mgr == 'systemd'
-
-  - name: check for anything running ceph
```

Comment:
If you see daemons still running after the package manager has removed/purged the packages, please file a ticket!